chat/views.py

```diff
@@ -1,17 +1,29 @@
+import json
+
 from django.contrib.auth.decorators import login_required
+from django.db.models import QuerySet
+from django.http import HttpRequest
 from django.shortcuts import render
+from typing import List
 
 from chat.models import Room
 
 
 @login_required
-def index(request):
+def index(request: HttpRequest):
     """
     Root page view. This is essentially a single-page app, if you ignore the
     login and admin parts.
     """
     # Get a list of rooms, ordered alphabetically
-    rooms = Room.objects.order_by("title")
+    rooms: QuerySet = Room.objects.order_by("title")
+    rooms_js: List[dict] = []
+
+    for room in rooms:
```

Comment:
I had this in mind too but I thought there's a shorter way of doing it.

Reply:
This the most dirty way of doing it.
I couldn't find a better way though, otherwise we should have just another endpoint